### PR TITLE
Added parameter to sample package and updated to latest Clion version

### LINKED ISSUE #70: Add Example of getting params to sample_package
- an example of getting params can be found here: https://github.com/UBC-Snowbots/Snowflake/blob/master/src/decision/src/gps_decision/gps_decision.cpp
- Just a simple example should do, perhaps replacing the character that is added to the end of the string?

### PR DESCRIPTION
**Pull request by ValRat**
Closes #70

### CODE REVIEW

File: src/sample_package/launch/rostest.launch [hunk 1]
```diff
@@ -1,3 +0,0 @@
-<launch>
-    <node
-</launch>
```
Comment (garethellis0) [inline src/sample_package/launch/rostest.launch]: Why are we deleting this file? Are we getting rid of the rostest stuff?

File: src/sample_package/src/MyNode.cpp [hunk 2]
```diff
@@ -14,6 +14,11 @@ MyClass::MyClass(int argc, char **argv, std::string node_name) {
     ros::NodeHandle nh;
     ros::NodeHandle public_nh("~");
 
+    // Obtains character from the parameter server (or launch file), sets '!' as default
+    std::string parameter_name = "my_node/character";
+    std::string default_character = "!";
+    nh.param(parameter_name, suffix, default_character);
```
Comment (garethellis0) [inline src/sample_package/src/MyNode.cpp]: We should be using `SB_getParam` here, from the kitchen sink library :wrench: . I've already added all the stuff to the cmake, should just be a matter of using it. Example [here](https://github.com/UBC-Snowbots/IGVC-2017/blob/master/src/decision/src/LidarDecision.cpp)

File: src/sample_package/launch/rostest.launch (hunk 1 above)
Comment (ValRat) [inline src/sample_package/launch/rostest.launch]: This was mistakenly committed in, rostests are defined under test/sample_package_test.test

File: src/sample_package/src/MyNode.cpp (hunk 2 above)
Comment (ValRat) [inline src/sample_package/src/MyNode.cpp]: Fixed :hammer_and_wrench:

File: src/sample_package/launch/rostest.launch (hunk 1 above)
Comment (ValRat) [inline src/sample_package/launch/rostest.launch]: This was mistakenly committed in, rostests are defined under test/sample_package_test.test

**Pull request merged**